packages/cli/src/commands/task.ts

```diff
@@ -156,28 +156,30 @@ const updateTaskMetadata = (
   }
 };
 
-function catFile(image: string, file: string): string {
+async function catFile(image: string, file: string): Promise<string> {
```

Comment:
It seems all these methods are also leftovers right? Can we delete all of them? 🔥

Reply:
For sure! Overlooked their removal! Thanks :)